File: JavaScript/DailyCodeChallenge/250816_areAnagrams.js
function areAnagrams(str1, str2) {

  const firstWord = str1.toUpperCase().replaceAll(" ","").split("").sort().join("")
  const secondWord = str2.toUpperCase().replaceAll(" ","").split("").sort().join("")


  return (firstWord == secondWord)
}


areAnagrams("School master", "The classroom")

/*
Anagram Checker
Given two strings, determine if they are anagrams of each other (contain the same characters in any order).

Ignore casing and white space.

Tests
Waiting:1. areAnagrams("listen", "silent") should return true.
Waiting:2. areAnagrams("School master", "The classroom") should return true.
Waiting:3. areAnagrams("A gentleman", "Elegant man") should return true.
Waiting:4. areAnagrams("Hello", "World") should return false.
Waiting:5. areAnagrams("apple", "banana") should return false.
Waiting:6. areAnagrams("cat", "dog") should return false.

*/

File: Python/Daily_Code_Challenge/250816-are-anagrams.py
def are_anagrams(str1, str2):

    first_word = sorted(str1.upper().replace(" ",""))
    second_word=sorted(str2.upper().replace(" ",""))

    return first_word==second_word




are_anagrams("School master", "The classroom")


'''
Anagram Checker
Given two strings, determine if they are anagrams of each other (contain the same characters in any order).

Ignore casing and white space.

Tests
Passed:1. are_anagrams("listen", "silent") should return true.
Passed:2. are_anagrams("School master", "The classroom") should return true.
Passed:3. are_anagrams("A gentleman", "Elegant man") should return true.
Passed:4. are_anagrams("Hello", "World") should return false.
Passed:5. are_anagrams("apple", "banana") should return false.
Passed:6. are_anagrams("cat", "dog") should return false.

'''